spring/src/test/java/io/seata/spring/annotation/MethodDescTest.java

```diff
@@ -37,6 +36,8 @@
     private static Class<?> targetClass = null;
     private static GlobalTransactional transactional = null;
 
+    private static final GlobalTransactionalInterceptor globalTransactionalInterceptor=new GlobalTransactionalInterceptor(null);
```

Comment:
The global variable declare in testGetAnnotation method?

Reply:
ok

Reply:
> The global variable declare in testGetAnnotation method?

PTAL @jsbxyyx